packages/compiler/src/constant_pool.ts

```diff
@@ -305,3 +313,8 @@ function invalid<T>(this: o.ExpressionVisitor, arg: o.Expression|o.Statement): n
 function isVariable(e: o.Expression): e is o.ReadVarExpr {
   return e instanceof o.ReadVarExpr;
 }
+
+function isLongStringExpr(expr: o.LiteralExpr): boolean {
+  return typeof expr.value === 'string' &&
+      expr.value.length > POOL_INCLUSION_LENGTH_THRESHOLD_FOR_STRINGS;
```

Comment:
Typically I would expect a "threshold" constraint to be satisfied on the boundary:

```suggestion
      expr.value.length >= POOL_INCLUSION_LENGTH_THRESHOLD_FOR_STRINGS;
```